python/ql/src/Expressions/WrongNameForArgumentInCall.ql

```diff
@@ -16,12 +16,22 @@
 import python
 import Expressions.CallArgs
 
-from Call call, FunctionObject func, string name
+from Call call, FunctionValue func, string name
 where
-    illegally_named_parameter_objectapi(call, func, name) and
-    not func.isAbstract() and
-    not exists(FunctionObject overridden |
-        func.overrides(overridden) and overridden.getFunction().getAnArg().(Name).getId() = name
-    )
+
+  illegally_named_parameter(call, func, name) and
+  not isAbstract(func) and
+  not exists(FunctionValue overridden |
+    func.overrides(overridden) and overridden.getScope().getAnArg().(Name).getId() = name
+  )
 select call, "Keyword argument '" + name + "' is not a supported parameter name of $@.", func,
-    func.descriptiveString()
+  func.descriptiveString()
+/*
```

Comment:
This comment should maybe just be deleted?

Reply:
Done!